assets/Discord-ServiceContract.md

```diff
@@ -0,0 +1,43 @@
+# Discord
+
+## Service Manager: _Chris Daniel_
+### Lifecycle: _Live_
+### Evolution: _Product_
+
+## Service Description
+_Discord is the easiest way to talk over voice, video, and text. Talk, chat, hang out, and stay close with other members of the Wardley Mapping Community._
```

Comment:
Some duplication here, and on the next line.  Consider something like:

The Foundation's __Discord service__ is is the easiest way to connect and collaborate with other members of the Wardley Mapping Community over __voice__, __video__ and __text__.  You can organise events, join voice & video calls, chat with groups & individuals in public & private channels, and share content.

This is a fundamental service that enables the communication required to achieve the Foundation's purpose.